releasenotes/notes/ancilla_allocation_no_cmap-ac3ff65b3639988e.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
+  - |
+    FullAncillaAllocation can now function with target's that do not have a
+    coupling map. In this case FullAncillaAllocation will add a number of
```

Comment:
```suggestion
    coupling map (typically because there are no 2 qubit gates in the :class:`~.Target`). In this case :class:`~.FullAncillaAllocation` will add
```